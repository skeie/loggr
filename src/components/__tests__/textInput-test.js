import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from '../TextInput';
test('Should render TextInput correctly', () => {
    const tree = renderer.create(
        <TextInput /> 
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
    