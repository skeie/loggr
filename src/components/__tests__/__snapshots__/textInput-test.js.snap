exports[`test Should render TextInput correctly 1`] = `
<TextInput
  autoFocus={undefined}
  keyboardType="default"
  onBlur={[Function]}
  onChangeText={[Function]}
  onFocus={[Function]}
  onSubmitEditing={undefined}
  placeholder={undefined}
  placeholderTextColor={undefined}
  returnKeyType="done"
  style={
    Array [
      Object {
        "color": "#9E9E9E",
      },
      undefined,
    ]
  }
  underlineColorAndroid="#E0E0E0" />
`;
